country-a-service/docker-compose.yml

```diff
@@ -3,9 +3,9 @@
 #
 networks:
   epps-backend:
-    ## Uncomment the following lines to use same network as OpenNCP:
-    # name: epps
-    # external: true
+    # Uncomment the following lines to use same network as OpenNCP:
+    name: epps
```

Comment:
NCP network is named "ncp" since wave 8, isn't it? We could potentially divide the network between NCP and country-A (epps), so only the node-a can access the country A services - but this will likely not work, unless epps is defined somewhere else.

Reply:
It's called `ncp` in the compose-file, but the external name is `epps` :)